src/KiNet/KinetPowerSupply.h

```diff
@@ -0,0 +1,59 @@
+//
+//  KinetPowerSupply.h
+//  ckLightsExample
+//
+//  Created by Alex Olivier on 11/17/14.
+//
+//
+
+#pragma once
+
+#include "KinetStrand.h"
+#include "asio/asio.hpp"
+
+namespace kinet {
+
+class KinetPowerSupply {
+
+public:
+  KinetPowerSupply(std::string ipAddress);
+  ~KinetPowerSupply();
+
+  void connectUDP();
+
+  std::shared_ptr<KinetStrand> addStrand(int iPort,
+                                         int iStrandSize = DEFAULT_STRAND_SIZE);
+  std::shared_ptr<KinetStrand> getStrand(int iPort);
+  std::vector<std::shared_ptr<KinetStrand>> getStrands();
+
+  void sendKinet();
+
+  std::string getIPAddress();
+
+protected:
+  std::vector<std::shared_ptr<KinetStrand>> strands;
+	std::shared_ptr<asio::ip::udp::socket> udpSocket;
+
+  std::string ipAddress;
+
+  bool loggedFailure = false;
+	bool udpSetup = false;
```

Comment:
Make formatting consistent within this file. These variable declarations are differently indented.